src/ui/widgets/project_card.py

```
@@ -0,0 +1,96 @@
+            open_count = self.project.get('open_count', 0)
+            last_opened = self.project.get('last_opened')
+
+            stats_parts = []
+            if open_count:
+                stats_parts.append(f"🔄 {open_count}")
+            if last_opened:
+                try:
+                    dt = datetime.fromisoformat(last_opened)
+                    time_ago = self._time_ago(dt)
+                    stats_parts.append(f"🕒 {time_ago}")
+                except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)